apps/console/src/features/identity-providers/components/wizards/apple/apple-authentication-provider-create-wizard.tsx

```diff
@@ -160,6 +160,7 @@ export const AppleAuthenticationProviderCreateWizard: FunctionComponent<
     const [ openLimitReachedModal, setOpenLimitReachedModal ] = useState<boolean>(false);
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
+    const [ current , setCurrent ] = useState<number>(0);
```

Comment:
the variable name `current` is mostly used with refs in React. Shall we rename this variable to be more descriptive, such as,

```suggestion
    const [ currentStepInSidePanelGuide , setCurrentStepInSidePanelGuide ] = useState<number>(0);
```

Please note that mentioned above is just a suggestion. Feel free to check and come up with more suitable variable name.